app/submitter/converter_v2.py

```diff
@@ -134,4 +136,22 @@ def get_payload_data(
             "lists": list_store.serialize(),
         }
 
+        if answer_codes := schema.json.get("answer_codes"):
+            get_filtered_answer_codes(answer_codes, data)
+
+        return data
+
     raise DataVersionError(schema.json["data_version"])
+
+
+def get_filtered_answer_codes(
+    answer_codes: Iterable[OrderedDict[str, Any]], data: dict
```

Comment:
answer_codes: Iterable Mapping?. Ordered dict isn't important for this?.

Reply:
I had to make it an iterable dict as otherwise mypy was complaining about the .extend method